Core/src/org/sleuthkit/autopsy/geolocation/datamodel/Waypoint.java

```diff
@@ -262,186 +262,6 @@ private static String getLabelFromArtifact(Map<BlackboardAttribute.ATTRIBUTE_TYP
 
         return attributeMap;
```

Comment:
import java.util.logging.Level and import org.sleuthkit.datamodel.SleuthkitCase are unused.